src/MicrosoftTeams.ts

```diff
@@ -417,26 +428,52 @@ namespace microsoftTeams
         {
             ensureInitialized(frameContexts.content, frameContexts.settings, frameContexts.remove);
 
+            authParams = authenticateParameters;
+
```

Comment:
THink about if authenticate is called twice. Shoudl we force-close the existing windows and/or call notifyFailure?